pkg/service/service.go

```diff
@@ -282,3 +283,66 @@ func RunService(c *cluster.Cluster, name string, token string, endpoint string,
 
 	return res.Body, nil
 }
+
+// RunService invokes a service synchronously (a Serverless backend in the cluster is required)
```

Comment:
Just a minor comment. This comment line is copied from previous method.
also you can change the variable "runServiceURL" name.